results-mieb/openai__clip-vit-base-patch16/57c216476eefef5ab752ec549e440a49ae4ae5f3/AROVisualAttribution.json

```diff
@@ -0,0 +1,21 @@
+{
+  "dataset_revision": "18f7e01358d91df599d723f00e16a18640e19398",
+  "evaluation_time": 339.30732822418213,
+  "kg_co2_emissions": null,
+  "mteb_version": "1.14.15",
+  "scores": {
+    "test": [
+      {
+        "accuracy": 0.0,
```

Comment:
Is this expected?

Reply:
yes it is expected for these 3 tasks (only one image per example so there's no image scores).

Reply:
Hmm not entirely clear from the name in the results file, might be worth renaming them to make it clear

Reply:
any suggestions? For this AbsTask, it is possible that both image and text have a score (such as [WinoGround](https://github.com/embeddings-benchmark/mteb/blob/mieb/results-mieb/openai__clip-vit-base-patch32/3d74acf9a28c67741b2f4f2ea7635f0aaf6f0268/Winoground.json)), in this case their `main score` can be `accuracy` (can perhaps be renamed to global accuracy?) which is when both text and image have are correct for an example. For these three tasks, only text has a score and their `main score` is set to text accuracy.

Reply:
We have used this approach over on main for the descriptive statistics.

https://github.com/embeddings-benchmark/mteb/blob/main/mteb/abstasks/AbsTaskPairClassification.py#L16

This allows us to add a description to each output. We could do a similar thing with the results. It can just be a [typeddict](https://peps.python.org/pep-0589/)